platform/x11/os_x11.cpp

```diff
@@ -1529,6 +1529,15 @@ void OS_X11::set_window_size(const Size2 p_size) {
 	int old_h = xwa.height;
 
 	Size2 size = p_size;
+
+	if (Math::is_nan(size.x)) {
+		WARN_PRINT("Incorrect width (NaN), using 0 instead.");
```

Comment:
```suggestion
		WARN_PRINT("Invalid width (NaN), using 0 instead.");
```

I think invalid sounds better than incorrect in this context.